cobalt/site/docs/gen/cobalt/doc/device_authentication.md

```diff
@@ -39,7 +39,7 @@ function to sign the message using the secret key.  This method is preferred
 since it enables implementations where the key exists only in secure hardware
 and never enters the system's main memory.  A reference implementation, which
 depends on BoringSSL exists at
-[starboard/linux/x64x11/internal/system_sign_with_certification_secret_key.cc](../../starboard/linux/x64x11/internal/system_sign_with_certification_secret_key.cc).
+[internal/starboard/linux/x64x11/internal/system_sign_with_certification_secret_key.cc](../../internal/starboard/linux/x64x11/internal/system_sign_with_certification_secret_key.cc).
```

Comment:
Let's change this to a code snipped

Reply:
Done. I don't know why we are editing the same .md doc in 2 different places now but oh well.